common/LabelNumber/LabelNumber.css

```diff
@@ -0,0 +1,32 @@
+.wrapper {
+    display: inline-block;
+    text-align: center;
+    height: 38px;
+    width: 70px;
+    background: #d8e3e2;
```

Comment:
There is no need to define the background here, it is specified by the parent. For preview purposes you can add a background colour in App.js

@arneskeie does this component need a fixed width?